common/test/acceptance/tests/studio/test_studio_library_container.py

```diff
@@ -240,13 +222,14 @@ def test_not_enough_children_blocks(self):
         expected_tpl = "The specified libraries are configured to fetch {count} problems, " \
```

Comment:
If we only allow one library now, should this be "The specified *library is configured* to fetch..."?